packages/ui/lit-spreadsheet/src/position.ts

```diff
@@ -0,0 +1,80 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+export type CellPosition = { i: number; j: number };
+
+export type CellRange = { from: CellPosition; to?: CellPosition };
+
+export const posEquals = (a: CellPosition | undefined, b: CellPosition | undefined) => a?.i === b?.i && a?.j === b?.j;
+
+export const colToA1Notation = (column: number): string => {
+  return (
+    (column >= 26 ? String.fromCharCode('A'.charCodeAt(0) + Math.floor(column / 26) - 1) : '') +
+    String.fromCharCode('A'.charCodeAt(0) + (column % 26))
+  );
+};
+
+export const rowToA1Notation = (row: number): string => {
+  return `${row + 1}`;
+};
+
+export const posToA1Notation = (column: number, row: number): string => {
+  const columnA1 = colToA1Notation(column);
+  const rowA1 = rowToA1Notation(row);
+  return `${columnA1}${rowA1}`;
+};
+
+export const posFromA1Notation = (notation: string): CellPosition => {
+  const match = notation.match(/([A-Z]+)(\d+)/);
+  if (!match) {
+    throw Error('[posFromA1Notation] No match.');
+  }
+  const column = match[1].split('').reduce((acc, c) => acc * 26 + c.charCodeAt(0) - 'A'.charCodeAt(0) + 1, 0) - 1;
+  const row = parseInt(match[2], 10) - 1;
+  return { i: column, j: row };
+};
+
+export const posFromNumericNotation = (notation: string): CellPosition => {
+  const [iStr, jStr] = notation.split(',');
+  if (!iStr || !jStr) {
+    throw Error('[posFromNumericNotation] Bad input');
```

Comment:
`throw new Error(`Invalid value: ${notation}``
The stack trace will show the function, and we'd want to know the value.

Reply:
I haven’t synced this implementation with yours yet, that should probably come as a follow-up.